src/main/java/com/intel/gkl/compression/IntelDeflater.java

```diff
@@ -55,12 +61,14 @@
     public synchronized boolean load(File tempDir) {
 
         if (!NativeLibraryLoader.load(tempDir, NATIVE_LIBRARY_NAME)) {
+            logger.warn(String.format("Unable to load %s", NATIVE_LIBRARY_NAME));
```

Comment:
same as above

Reply:
I can remove this duplicate since its already covered in NativeLibraryLoader class